core/discord_api.py

```diff
@@ -0,0 +1,62 @@
+import requests
+
+from gwardia_hub.settings import CLIENT_ID, CLIENT_SECRET, DEBUG, GUILD_ID
+
+API_ENDPOINT = "https://discord.com/api/v10"
+REDIRECT_URI = (
+    "http://localhost:8000/auth/discord/code"
+    if DEBUG
+    else "https://gwardiahub.fly.dev/auth/discord/code"
+)
+DISCORD_TOKEN_COOKIE = "discord_access_token"
+DISCORD_REFRESH_COOKIE = "discord_refresh_token"
+
+
+class UnauthorizedException(Exception):
+    pass
+
+
+def code_to_token(code):
```

Comment:
```suggestion
def validate_auth_code(code):
```

Reply:
But we aren't validating anything.
Maybe `exchange_code`?